src/config.py

```diff
@@ -49,6 +49,27 @@ class PolicyEvaluationChecks(TypedDict):
 	evaluation_checks: list[JsonPathCheck]
 
 
+class Suggestion(TypedDict):
+	"""
+	A suggestion for a modification when a rule matches.
+	"""
+
+	pattern: str
+	"""
+	A pattern to match in the diff.
+	"""
+
+	pattern_regex: re.Pattern
+	""" Internal """
+
+	replacement: str
+	"""
+	The replacement for the pattern.
+	Python regex style substitution is supported.
+	For example: TODO Add examples.
```

Comment:
Do you want to address this TODO in this PR or is this intentionally left for later?

Reply:
Yeah I should come some example from the README.md